### PR TITLE
Sp/#275 ev input evcs input

### PR DESCRIPTION
**Pull request by sebastian-peter**
Solves #275

### CODE REVIEW

File: src/test/groovy/edu/ie3/datamodel/io/factory/input/participant/EvcsInputFactoryTest.groovy [hunk 1]
```diff
@@ -44,9 +43,9 @@ class EvcsInputFactoryTest extends Specification  implements FactoryTestHelper {
 			"operatesuntil"   : "2019-12-31T23:59:00+01:00[Europe/Berlin]",
 			"id"              : "TestID",
 			"qcharacteristics": "cosPhiFixed:{(0.0,1.0)}",
-			"type"          : "Household",
-			"chargingpoints"         : "4",
-			"cosphirated"          : "0.95",
+			"type"            : "Household",
```
Comment (ckittl) [inline src/test/groovy/edu/ie3/datamodel/io/factory/input/participant/EvcsInputFactoryTest.groovy:46]: Is there a place, where the parsing of _all_ permissible input it tested?

**Review by ckittl — APPROVED** [review]
Looks fine to me! Was just wondering, if the naming of _evcs_ (electrical vehicle charging station) is still, what we would like to go for? In our discussion we also talked a lot about _lis_ as loading infrastructure.

File: src/test/groovy/edu/ie3/datamodel/io/factory/input/participant/EvcsInputFactoryTest.groovy (hunk 1 above)
Comment (sebastian-peter) [reply src/test/groovy/edu/ie3/datamodel/io/factory/input/participant/EvcsInputFactoryTest.groovy:46]: Yes, that's what happens in [ChargingPointTypeUtilsTest](https://github.com/ie3-institute/PowerSystemDataModel/blob/dev/src/test/groovy/edu/ie3/datamodel/models/input/system/type/chargingpoint/ChargingPointTypeUtilsTest.groovy)

**Comment by sebastian-peter** [comment]
> Looks fine to me! Was just wondering, if the naming of _evcs_ (electrical vehicle charging station) is still, what we would like to go for? In our discussion we also talked a lot about _lis_ as loading infrastructure.

Good question. It was confusing me in the beginning. EVCS sounds more intuitive to me, but probably @johanneshiry knows more.

**Review by johanneshiry — APPROVED** [review]

**Pull request merged**